lib/utapi/utilities.js

```diff
@@ -103,9 +102,10 @@ export function listMetrics(metricType) {
         .option('-p, --port <port>', 'Port of the server')
         .option('--ssl', 'Enable ssl')
         .option('-v, --verbose')
+        .option('-r, --recent', 'List metrics of the latest 15 minute interval')
```

Comment:
It should be the nearest 15 min interval, ie 2:15 - current if it is 2:20.
@bennettbuchanan Can you update the wording to reflect _starting from the nearest 15 minute interval to the current time_.